Makefile

```diff
@@ -38,60 +38,34 @@ technical-docs:  ## build the technical docs
 guides:	swagger ## build the developer guide docs
 	cd docs/guides; make clean html
 
-# (IS_OPENEDX_TRANSLATIONS_WORKFLOW) is set to "yes" in the `extract-translation-source-files` GitHub actions
-# workflow on the `openedx-translations` repository. See (extract translation source files) step here:
-# https://github.com/openedx/openedx-translations/blob/main/.github/workflows/extract-translation-source-files.yml
-# Related doc: https://docs.openedx.org/en/latest/developers/how-tos/enable-translations-new-repo.html
-ifeq ($(IS_OPENEDX_TRANSLATIONS_WORKFLOW),yes)
 extract_translations: ## extract localizable strings from sources
 	i18n_tool extract --no-segment -v
 	cd conf/locale/en/LC_MESSAGES && msgcat djangojs.po underscore.po -o djangojs.po
 	cd conf/locale/en/LC_MESSAGES && msgcat django.po wiki.po edx_proctoring_proctortrack.po mako.po -o django.po
 	cd conf/locale/en/LC_MESSAGES && rm wiki.po edx_proctoring_proctortrack.po mako.po underscore.po
-else
-extract_translations: ## extract localizable strings from sources
-	i18n_tool extract -v;
-endif
-
-push_translations: ## push source strings to Transifex for translation
-	i18n_tool transifex push
 
 pull_plugin_translations:  ## Pull translations for edx_django_utils.plugins for both lms and cms
-	rm -rf conf/plugins-locale/plugins  # Clean up existing atlas translations
-	mkdir -p conf/plugins-locale/plugins
 	python manage.py lms pull_plugin_translations --verbose $(ATLAS_OPTIONS)
 	python manage.py lms compile_plugin_translations
 
 pull_xblock_translations:  ## pull xblock translations via atlas
-	rm -rf conf/plugins-locale/xblock.v1  # Clean up existing atlas translations
-	rm -rf lms/static/i18n/xblock.v1 cms/static/i18n/xblock.v1  # Clean up existing xblock compiled translations
 	python manage.py lms pull_xblock_translations --verbose $(ATLAS_OPTIONS)
 	python manage.py lms compile_xblock_translations
 	python manage.py cms compile_xblock_translations
 
-pull_translations: ## pull translations from Transifex
-	git clean -fdX conf/locale conf/plugins-locale/studio-frontend
-ifeq ($(OPENEDX_ATLAS_PULL),)
-	i18n_tool transifex pull
-	i18n_tool extract
-	i18n_tool dummy
-	i18n_tool generate --verbose 1
-	git clean -fdX conf/locale/rtl
-	git clean -fdX conf/locale/eo
-	i18n_tool validate --verbose
-else
+pull_translations: ## pull translations via atlas
+	# Clean up the existing translations
+	git clean -fdX conf/locale conf/plugins-locale */static/js/i18n/ */static/js/xblock.v1-i18n/
```

Comment:
This line is causing some problems -- it deletes any virtualenv that's maintained inside the repo (`.venv-3.8/` in my case) as well as the `Open_edX.egg-info/` created when installing base requirements. The former causes the command to fail entirely, and the latter produces the following error:

```
RuntimeError: Model class openedx.core.djangoapps.content_libraries.models.ContentLibrary doesn't declare an explicit app_label and isn't in an application in INSTALLED_APPS.
make[1]: *** [Makefile:52: pull_xblock_translations] Error 1
```

Can we use a different command? Or is this even needed at all. It seems to me that a git pull should remove the files just fine.

Reply:
It appears that if one of the path arguments to `git clean` doesn't exist **and** contains a globbing character (e.g. `*/static/js/xblock.v1-i18n/`) then the command also removes some things outside of the given paths.

Reply:
@timmc-edx sorry about that. I don't fully understand why `git clean` is behaving like this, so I've gone and refactored the command to be more explicit and safe:

 - https://github.com/openedx/edx-platform/pull/34380